x-pack/plugins/security_solution/public/resolver/mocks/resolver_tree.ts

```diff
@@ -137,29 +130,19 @@ function withRelatedEventsOnOrigin(
       byCategory,
     },
   };
-  for (const [category, type] of relatedEventsToAddByCategoryAndType) {
-    events.push({
-      '@timestamp': 1,
-      event: {
-        kind: 'event',
-        type,
-        category,
-        id: 'xyz',
-      },
-      process: {
-        entity_id: treeToAddRelatedEventsTo.entityID,
-      },
-    });
+  for (const event of events) {
     stats.events.total++;
-    stats.events.byCategory[category] = stats.events.byCategory[category]
-      ? stats.events.byCategory[category] + 1
-      : 1;
+    for (const category of eventModel.eventCategory(event)) {
```

Comment:
:grey_question: Do we even need to do this anymore since we're not doing aggregate counts? Seems like it should come from a data layer or something now.

Reply:
no I think I goofed something up.

Reply:
no I think I goofed something up.